File: rest-api-crud/src/main/java/com/shy/restapicrud/Provider.java
package com.shy.restapicrud;

import javax.persistence.Entity;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Entity
@Data
@AllArgsConstructor
@NoArgsConstructor
public class Provider {
	
	String Cnpj;
	String Name;
	String RegisterDate;
}

File: CRUD/CRUD.Api/Model/Provider.cs
using System.ComponentModel.DataAnnotations;

namespace CRUD.Api
{
    public class Provider
    {   
        [Key]
        public string Cnpj { get; set; } = string.Empty;
        public string Name { get; set; } = String.Empty;
        public string RegisterDate { get; set; } = String.Empty;
    }
}
